src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,33 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static int MIN_PASSWORD_LENGTH = 6;
+    private static int MIN_AGE = 18;
+    private StorageDao storageDao;
+
+    public RegistrationServiceImpl() {
+        this.storageDao = new StorageDaoImpl();
+    }
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null
+                || user.getLogin() == null
+                || user.getPassword() == null
+                || user.getAge() == null) {
+            throw new NullPointerException("Null input");
+        }
```

Comment:
You can split null checks for more informative exception messages.
Just my thought, don't know if it's required)